lib/iris/fileformats/netcdf.py

```diff
@@ -902,11 +919,16 @@ def write(self, cube, local_keys=None, unlimited_dimensions=None,
             on `cube.data` and possible masking. For masked data, `fill_value`
             is taken from netCDF4.default_fillvals. For more control, pass a
```

Comment:
I think this needs a tweak as it seems to say that if 'packing' is used then 'fill_value' would be ignored.
Whereas, the new note for 'fill_value' itself implies that it *will* be used for packing, if given, which I think is correct.